src/content/reference/manifest.mdx

```diff
@@ -560,6 +560,47 @@ selector:
   app.kubernetes.io/name: vote
 ```
 
+### services ([object], optional)
+
+A list of other containers (other than the main development container) that you want to put on developer mode.
+The difference with the main development container and the ones defined in this section is that the main development container gives you an interactive session, where you can, for example, execute a shell.
+The containers defined in this section run in detached mode, but they can also mount your local folders in a particular path.
+
+For example, to put a Django application on developer mode you could use the following manifest:
```

Comment:
Most people wont'be familiar with the Django application model, I think you need to explain it a bit more. 

Maybe something like this?

```suggestion
For example, imagine that you have a python-based application with an API and a Worker service. If you define the manifest as shown below, running `okteto up` would give you a remote terminal into the web development container, while synchronizing your changes with both web and worker.
```